ftw/recipe/deployment/pack.py

```diff
@@ -20,7 +20,8 @@ def create_pack_script(recipe):
             blob_storage = recipe.buildout[zeo_part].get(
                 'blob-storage', 'blobstorage')
             blob_storage = os.path.join(recipe.buildout_dir, 'var', blob_storage)
-        storages.append((file_storage, blob_storage))
+        zeopack_name = recipe.buildout[zeo_part].get('zeopack-script-name', 'zeopack')
```

Comment:
It looks to me like with the 
```
file_storage = recipe.buildout[zeo_part].get('storage-number', '1')
```
above (line 17) there is a default that assumes there is only one storage. You're not setting `storage-number` for the second zeo in any of the examples or in the edubs setup. Does this really work properly?